webapp/channels/src/packages/mattermost-redux/src/actions/emojis.ts

```diff
@@ -69,16 +72,46 @@ export function getCustomEmojiByName(name: string): ActionFunc {
 }
 
 export function getCustomEmojisByName(names: string[]): ActionFunc {
-    return async (dispatch: DispatchFunc) => {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
         if (!names || names.length === 0) {
             return {data: true};
         }
 
-        const promises: Array<Promise<ActionResult|ActionResult[]>> = [];
-        names.forEach((name) => promises.push(dispatch(getCustomEmojiByName(name))));
+        let data;
+        try {
+            data = await Client4.getCustomEmojisByNames(names);
```

Comment:
Should we handle here batches for cases we have more than 200 emojis?

Reply:
Good catch. I'll add that